docs/understanding-tracking-design/managing-your-data-structures/ui/index.md

```diff
@@ -12,7 +12,15 @@ To create a new [Data Structure](/docs/understanding-your-pipeline/schemas/index
 
 ![](images/image-1.png)
 
-When creating a new data structure, you can select from two options: **Create using builder** or **Create using JSON**.
+Enterprise customers can select from two options: **Create using builder** or **Create using JSON**. The [Data Structures Builder](./builder/) only supports the following types: 
+
+- string
+- enumerated list
+- integer 
+- decimal
+- boolean
+
+For more complex data structures that require heavy nesting and more advanced data types, use the [JSON Editor](./json-editor/). BDP Cloud accounts will be automatically directed to the [Data Structures Builder](./builder/).
```

Comment:
```suggestion
For more complex data structures that require heavy nesting and more advanced data types, use the [JSON Editor](./json-editor/index.md). BDP Cloud accounts will be automatically directed to the [Data Structures Builder](./builder/index.md).
```

Reply:
In addition, I recommend to switch to absolute links. With relative links, if we want to move the inner pages automatically, it will be not possible to update the links automatically as well. (At least, not with `move.sh`.)